README.md

```diff
@@ -1,13 +1,3 @@
-Forward...
-
-# Hacking RubyGems
-
-
-- http://blog.honeybadger.io/stop-using-rubygemsorg-in-production/
-- https://speakerdeck.com/benjaminleesmith/hacking-with-gems-rubynation-2013
```

Comment:
probably should read this on why a gem on rubygems.org has no necessary relationship to a repo on github